pages.bn/android/getprop.md

```diff
@@ -0,0 +1,32 @@
+# getprop
+
+> অ্যান্ড্রয়েড সিস্টেম বৈশিষ্ট্য সম্পর্কে তথ্য দেখান।
+> আরও তথ্য পাবেন: <https://manned.org/getprop>.
+
+- অ্যান্ড্রয়েড সিস্টেম বৈশিষ্ট্য সম্পর্কে তথ্য প্রদর্শন করুন:
+
+`getprop`
+
+- একটি নির্দিষ্ট সম্পত্তি সম্পর্কে তথ্য প্রদর্শন করুন:
+
+`getprop {{prop}}`
```

Comment:
```suggestion
- একটি নির্দিষ্ট সম্পত্তি সম্পর্কে তথ্য প্রদর্শন করুন:

`getprop {{সম্পত্তি}}`
```

Optional fix (error source: original page): unexpected untranslated placeholder, `property` can be used as a translation source.